theme: added new theme Negligible

SagarYadav17 opened the pull request:
### Prerequisites

- [ ] I have read and understand the `CONTRIBUTING` guide
- [ ] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### Description

[Description of the change]

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

sagaryadav17-bot committed:
theme: added new theme Negligible

sagaryadav17-bot committed:
theme: added new theme Negligible

JanDeDobbeleer commented on themes/negligible.omp.json:
```diff
@@ -0,0 +1,82 @@
+                {
+                    "type": "os",
+                    "style": "powerline",
+                    "foreground": "cyan"
```
It's prettier when this one has the `"prefix": ""` property. Not sure if this is on purpose, if so I'm inclined to merge, just my two cents as without that there's a space before the OS icon starts.

SagarYadav17 replied:
Thanks for mentioning that, I will fix that

sagaryadav17-bot committed:
fix: space between OS and PATH in negligible theme

JanDeDobbeleer commented:
@SagarYadav17 can you squash those two commits into 1 and force push your branch? I'll merge right after that.

sagaryadav17-bot committed:
theme: changed exit code representation

Now in negligible theme,
- the arrow will change its color to red if any error occurs.
- whitespace after an error occurs is removed.

SagarYadav17 commented:
@JanDeDobbeleer sorry but I don't know how to do that. Any guidance.

JanDeDobbeleer commented:
The easiest way, given no changes have been made to main on our end is to use the following commands:

```
git rebase -i main
```

That opens your git editor and you can mark the `fix:` commit as `fixup` and close the editor. This will merge that commit in the previous one and keep the first commits message. You can now force push your branch:

```
git push --force
```

Done 🙂

sagaryadav17-bot committed:
Merge branch 'main' of github.com:SagarYadav17/oh-my-posh3 into pr/SagarYadav17/336

SagarYadav17 commented:
@JanDeDobbeleer Done 👍🏼

sagaryadav17-bot committed:
Merge branch 'main' into main

JanDeDobbeleer commented:
Merged this manually.
